operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -418,7 +409,7 @@ func (rc *ReconciliationContext) CheckPodsReady() result.ReconcileResult {
 
 	// step 2 - get one node up per rack
 
-	rackWaitingForANode, err := rc.startOneNodePerRack(seedCount)
+	rackWaitingForANode, err := rc.startOneNodePerRack(endpointData, seedCount)
```

Comment:
I'm not sure about this change

Reply:
How come? We have the potential to use the endpoint data in multiple checks (stuck pods, for example) so I extracted out to a single api call just like the podlist

Reply:
ok this makes more sense to me now